Makefile

```
@@ -87,7 +87,7 @@ test/integration:
 # Run commands
 
 .PHONY: run/pipecd
-run/pipecd: BUILD_VERSION ?= $(shell git describe --tags --always --dirty --abbrev=7)
+run/pipecd: BUILD_VERSION = "$(shell git describe --tags --always --abbrev=7)-$(shell date +%s)"
```

```suggestion
run/pipecd: BUILD_VERSION ?= "$(shell git describe --tags --always --abbrev=7)-$(shell date +%s)"
```
Should keep `?=` since the we may want to specified BUILD_VERSION directly in some cases